cmd/root.go

```diff
@@ -84,6 +84,7 @@ func initConfig() {
 	}
 
 	viper.SetDefault(configuration.ViperKeyLogLevel, "info")
+	viper.SetDefault(configuration.ViperKeyCGroupsV1AutoMaxProcsEnabled, false)
```

Comment:
No need to set this when the default is configured in the config schema :)

Reply:
removed this line